draft-packages/popover/KaizenDraft/Popover/Popover.tsx

```diff
@@ -1,255 +1,35 @@
-import { Icon } from "@kaizen/component-library"
-import closeIcon from "@kaizen/component-library/icons/close.icon.svg"
-import negativeIcon from "@kaizen/component-library/icons/exclamation.icon.svg"
-import informativeIcon from "@kaizen/component-library/icons/information.icon.svg"
-import positiveIcon from "@kaizen/component-library/icons/success.icon.svg"
+import React, { useEffect } from "react"
+import { PopoverLegacy, LegacyPopoverProps } from "./PopoverLegacy"
+import { PopoverModern, ModernPopoverProps } from "./PopoverModern"
 
-import classNames from "classnames"
-import * as React from "react"
+export type Props = LegacyPopoverProps | ModernPopoverProps
 
-import styles from "./styles.scss"
+type PopoverType = React.FunctionComponent<Props>
 
-export interface Props {
-  readonly id?: string
-  readonly automationId?: string
-  readonly visible?: boolean
-  readonly onClose?: (event: React.MouseEvent<HTMLButtonElement>) => any
-  readonly variant?: Variant
-  readonly side?: Side
-  readonly size?: Size
-  readonly position?: Position
-  readonly heading?: string
-  readonly dismissible?: boolean
-  readonly singleLine?: boolean
-  readonly children: React.ReactNode
-  readonly boxOffset?: BoxOffset
```

Comment:
cc @stephwestgarth, this property will be removed since popper now takes care of the offset for us. I will this up on perf-ui. LMK if any issues.